gnus-recent.el

```diff
@@ -64,28 +64,29 @@
   "Face used for dates in the recent article list."
   :group 'gnus-recent)
 
+(defun gnus-recent-date-format (date)
+  "Convert the DATE to 'YYYY-MM-D HH:MM:SS a' format."
+  (condition-case ()
+      (format-time-string "%F %T %a" (gnus-date-get-time date))
```

Comment:
If we're changing it anyway, maybe it should be a defcustom?

Reply:
Yeah! 
but can it wait, until a number of other defcustoms are created ?
It seems, to me, like overkill to introduce only one customization for a small library like this. 
Up to you!

Reply:
sure